spec/models/firm_factory_spec.rb

```diff
@@ -0,0 +1,189 @@
+RSpec.describe 'Firm factory' do
+  subject do
+    begin
+      FactoryGirl.create(factory)
+    rescue ActiveRecord::RecordInvalid
```

Comment:
@asmega some fixtures need to be created in an invalid state and therefore cannot be persisted. So yes, we do want these tests to fall back to `build`.